### PR TITLE
Adds button to organizations list for prefilled distributions

### LINKED ISSUE #931: Add a button component that can open the widget with a given distribution
Create a component that has a button that opens the widget with a pre-filled distribution, and optionally a pre-filled sum.
Add this component to organizations in organizations list, so that we can add a button directly on organizations that opens the widget with a pre-filled distribution.

### PR DESCRIPTION
**Pull request by fellmirr**
Adds a button in the organizations list for opening a prefilled distribution

- closes #931 

---

Tested on devices

- [ ] Desktop 💻
- [ ] Mobile 📱

Tests

- [x] All tests are running ✔️
- [x] Test are updated 🧪
- [x] Code Review 👩‍💻
- [x] QA 👌

Checkpoints

_Check these to flag for a more thurough review, as they could be potentially breaking changes_

- [x] Packages updated
- [ ] Other infrastructure updated (such as node version or similar)

⏲️ Time spent on CR:

⏲️ Time spent on QA:

### CODE REVIEW

**Comment by fellmirr** [comment]
@EliseAndresen Ready for QA :)

**Comment by EliseAndresen** [comment]
Kan vi ha en lik knapp for "GW All Grants Fund"?

File: components/profile/layout/layout.tsx [hunk 1]
```diff
@@ -118,9 +121,13 @@ export const ProfileLayout = withStaticProps(async ({ preview }: { preview: bool
                   toastStyle={{ borderRadius: 0, background: "white", color: "black" }}
                 />
                 {isPreview && <PreviewBlock />}
-                <WidgetContext.Provider value={[widgetOpen, setWidgetOpen]}>
+                <WidgetContext.Provider value={[widgetContext, setWidgetContext]}>
```
Comment (adamalfredsson) [inline components/profile/layout/layout.tsx:124]: Recommended to use a stable context provider value, with `useMemo()`

File: components/shared/components/Widget/components/Widget.tsx [hunk 2]
```diff
@@ -216,6 +225,54 @@ export const Widget = withStaticProps(async ({ preview }: { preview: boolean })
     dispatch(fetchReferralsAction.started(undefined));
   }, [dispatch]);
 
+  useEffect(() => {
```
Comment (adamalfredsson) [inline components/shared/components/Widget/components/Widget.tsx:228]: Perhaps we should make sure to only prefill once. For instance, if you use open a prefilled widget, change distribution, close and reopen the widget (with the GiveButton), the prefilled distribution will be applied again overwriting any previous changes.

Comment (fellmirr) [reply components/shared/components/Widget/components/Widget.tsx:228]: Yes, this is a bug! Should run the effect on pre-filled, not the entire widget context. Fixed now and added tests to cover this scenario. If you choose a pre-filled option, change the distribution, and then choose the same pre-fill button, it won't update the distribution, but I think that is a reasonably niche scenario. Might be worth looking out for in hotjar though @EliseAndresen.

**Pull request merged**